warehouse/email/__init__.py

```diff
@@ -201,6 +201,48 @@ def send_added_as_collaborator_email(request, user, *, submitter, project_name,
     return {"project": project_name, "submitter": submitter.username, "role": role}
 
 
+@_email("removed-package")
+def send_removed_package_email(
+    request, user, *, project_name, submitter_name, submitter_role, recipient_role
+):
+    recipient_role_descr = "an owner"
+    if recipient_role == "Maintainer":
+        recipient_role_descr = "a maintainer"
+
+    return {
+        "project": project_name,
+        "submitter": submitter_name,
+        "submitter_role": submitter_role,
+        "recipient_role_descr": recipient_role_descr,
+    }
+
+
+@_email("removed-package-release")
+def send_removed_package_release_email(
```

Comment:
I think it'd be OK to drop the "package" part here:

```python
@_email("removed-release")
def send_removed_release_email(
```

Reply:
Could the signature for this be reduced to:

```
(request, user, *, release, submitter_name, submitter_role, recipient_role)
```

`project_name` can be retrieved from `release.project.name` and `release_date` can be retrieved from `self.release.created`